packages/ui-components/src/components/Button/Button.component.js

```diff
@@ -130,6 +130,7 @@ export const Button = React.forwardRef(
     ) : icon ? (
       <Icon
         icon={icon}
+        title={title}
```

Comment:
Would be possible to introduce the tests for this component as well?

Reply:
I added a unit test for this case now.

Reply:
To ensure the displayed title matches the button title, you should use `titleValue` instead of the raw prop value.

```suggestion
        title={titleValue}
```

Reply:
Yes, you are right, the titleValue is preferable.